src/utils.ts

```diff
@@ -26,16 +26,29 @@ function loadTsFile(filename: string, cwd: string, tsConfig?: any): any {
   let { data, path } = joycon.loadSync([filename], cwd)
   if (path && data) {
     if (tsConfig) {
-      data = deepmerge(tsConfig, data)
+      data = deepmerge(data, tsConfig)
     }
-    const { extends: _extends } = data
+    let { extends: _extends } = data
     if (_extends) {
       delete data.extends
-      return loadTsFile(path, cwd, data)
```

Comment:
The existing code simply calls its self again but deletes the `extends` property doesn't actually try to access the extended tsconfig.